src/commands/help.ts

```diff
@@ -0,0 +1,8 @@
+export const help: GlobalCommand = {
+	name: 'help',
+	description: 'Prints the list of commands',
+	requiresGuild: false,
+	async execute({ reply }) {
+		await reply('Hello, world!');
```

Comment:
```suggestion
		// TODO: finish this command
		await reply('Hello, world!');
```
Just for the todo tree :)